api/src/test/java/org/openmrs/module/metadatadeploy/handler/impl/ConceptDeployHandlerTest.java

```diff
@@ -56,223 +56,192 @@
-
-        Concept created = conceptService.getConceptByUuid(UUID);
-        assertThat(created.getNames().size(), is(1));
-        assertThat(created.getName(), allOf(
-                hasProperty("name", is("History of present illness")),
-                hasProperty("locale", is(Locale.ENGLISH)),
-                hasProperty("localePreferred", is(true)),
-                hasProperty("conceptNameType", is(ConceptNameType.FULLY_SPECIFIED)),
-                hasProperty("tags", containsInAnyOrder(Arrays.asList(
-                                hasProperty("tag", is("refapp-preferred"))
-                        )
-                ))));
-        assertThat(created.getDescriptions().size(), is(1));
-        assertThat(created.getDescription(), allOf(
-                hasProperty("description", is("How the patient describes it")),
-                hasProperty("locale", is(Locale.ENGLISH))));
-    }
-
-    @Test
-    public void testCreateNumeric() throws Exception {
-        ConceptNumeric numeric = new ConceptNumeric();
-        numeric.setUuid(UUID);
-        numeric.setDatatype(conceptService.getConceptDatatypeByName("Numeric"));
-        numeric.setConceptClass(existing(ConceptClass.class, "a82ef63c-e4e4-48d6-988a-fdd74d7541a7"));
-        numeric.addName(new ConceptName("Favorite single digit number", Locale.ENGLISH));
-        numeric.addDescription(new ConceptDescription("1-10", Locale.ENGLISH));
-        numeric.setLowAbsolute(1d);
-        numeric.setHiAbsolute(10d);
-        numeric.setPrecise(false);
-        deployService.installObject(numeric);
-
-        ConceptNumeric created = (ConceptNumeric) conceptService.getConceptByUuid(UUID);
-        assertThat(created.getLowAbsolute(), is(1d));
-        assertThat(created.getHiAbsolute(), is(10d));
-        assertThat(created.getPrecise(), is(false));
-        assertThat(created.getName().getName(), is("Favorite single digit number"));
-    }
-
-    @Test
-    public void testCreateNumericUsingBuilder() throws Exception {
-        deployService.installObject(new ConceptNumericBuilder(UUID)
-                .datatype(conceptService.getConceptDatatypeByName("Numeric"))
-                .conceptClass(existing(ConceptClass.class, "a82ef63c-e4e4-48d6-988a-fdd74d7541a7"))
-                .name(new ConceptName("Favorite single digit number", Locale.ENGLISH))
-                .description(new ConceptDescription("1-10", Locale.ENGLISH))
-                .lowAbsolute(1d)
-                .hiAbsolute(10d)
-                .precise(false)
-                .build());
-
-        ConceptNumeric created = (ConceptNumeric) conceptService.getConceptByUuid(UUID);
-        assertThat(created.getLowAbsolute(), is(1d));
-        assertThat(created.getHiAbsolute(), is(10d));
-        assertThat(created.getPrecise(), is(false));
-        assertThat(created.getName().getName(), is("Favorite single digit number"));
-    }
-
-    @Test
-    public void testUpdate() throws Exception {
```

Comment:
Did you make any changes in this method? If so, did you rewrite the whole method?

Reply:
i did not make any changes here and have removed the file that i had pushed